From 9d50c99de68a8cef82055339a51ad4a7bafe860d Mon Sep 17 00:00:00 2001
From: Samreen Nilofer <samreen.nilofer@intel.com>
Date: Wed, 14 Jan 2015 14:08:33 +0530
Subject: [PATCH 04/15] ASoC: Intel: Enable the IPC dump message

Issue: GMINL-6292

print_hex_dump_bytes() is a debug print and will be enabled only
then debug logs are enabled.

Disabling the Macro to support the dump.

Change-Id: If36a71e9f157a05f18b798763e85c75427dffea6
Signed-off-by: Samreen Nilofer <samreen.nilofer@intel.com>
Signed-off-by: Fabien Marotte <fabien.marotte@intel.com>
---
 sound/soc/intel/platform-libs/controls_v2_dpcm.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/sound/soc/intel/platform-libs/controls_v2_dpcm.c b/sound/soc/intel/platform-libs/controls_v2_dpcm.c
index 8ea4be9..e7fda03 100644
--- a/sound/soc/intel/platform-libs/controls_v2_dpcm.c
+++ b/sound/soc/intel/platform-libs/controls_v2_dpcm.c
@@ -33,8 +33,6 @@
 #include "controls_v2_dpcm.h"
 #include "sst_widgets.h"
 
-#define DUMP_LPE_BUFFERS 0
-
 static inline void sst_fill_byte_control(char *param,
 					 u8 ipc_msg, u8 block,
 					 u8 task_id, u8 pipe_id,
@@ -56,10 +54,8 @@ static inline void sst_fill_byte_control(char *param,
 	byte_data->len = len;
 	memcpy(byte_data->bytes, cmd_data, len);
 
-#if	DUMP_LPE_BUFFERS
 	print_hex_dump_bytes("writing to lpe: ", DUMP_PREFIX_OFFSET,
 			     byte_data, len + sizeof(*byte_data));
-#endif
 }
 
 unsigned int sst_soc_read(struct snd_soc_platform *platform,
-- 
1.9.1

